packs/bit-bucket/requirements.txt

```diff
@@ -0,0 +1 @@
+requests>=2.6.0
```

Comment:
What was missing in the upstream module that is not in the API? Looks like several good actions in the official API that aren't quite in here yet.